File: CPP/ArrayExample.cpp
#include<iostream>
using namespace std;
int main()
{
	int nos[5];
	cout<<"Please enter 5 integers below:"<<endl;
	for(int i=0; i<5; i++)
	{
		cin>>nos[i];
	}
	cout<<"The integers entered are as follows: "<<endl;
	for(int i=0; i<5; i++)
	{
		cout<<nos[i]<<endl;
	}
	return 0;
}

File: Java/ArrayExample.java
import java.util.Scanner;
class ArrayExample {
	public static void main(String[] args)
	{
		Scanner scan = new Scanner(System.in);
		int[] nos = new int[5];
		System.out.println("Please enter 5 integers below:");
		for(int i=0; i<5; i++)
		{
			nos[i] = scan.nextInt();
		}
		System.out.println("The integers entered are as follows:");
		for(int i=0; i<5; i++)
		{
			System.out.println(nos[i]);
		}
	}
}
